### ISSUE TITLE
Pod already created and exited but its status remains ContainerCreating

### ISSUE THREAD

**Issue by zionwu** [kubernetes/kubernetes#98661]
**What happened**:
We have a k8s cluster for deep learning training.
Recently,  We find that sometimes there is pod stuck on `ContainerCreating` status.  
However, the events indicates the pod is created successfully:
```
Events:
  Type     Reason         Age                 From                 Message
  ----     ------         ----                ----                 -------
  Warning  Unschedulable  19m (x39 over 25m)  kube-batch           1/1 tasks in gang unschedulable: 0 nodes are available
  Normal   Pulled         18m                 kubelet, 10.196.2.7  Container image "registry.xxx.xyz/xxx/training_sdk_gpu:v0.2.9" already present on machine
  Normal   Created        18m                 kubelet, 10.196.2.7  Created container tensorflow
  Normal   Started        18m                 kubelet, 10.196.2.7  Started container tensorflow
```
The containers of pod is already created on the host:
```
# docker ps -a |grep 20210201110655300
dda3ddcaa516        "registry.xxx.xyz/xxx/training_sdk_gpu:v0.2.9          "/bin/bash -c 'echo …"   18 minutes ago         Exited (1) 18 minutes ago                          k8s_tensorflow_iac-20210201110655300-worker-0_11071760_b4e541e4-643a-11eb-9823-f403434ab74c_0
739f751212a5        rancher/pause:3.1                                    "/pause"                 18 minutes ago         Up 18 minutes                                k8s_POD_iac-20210201110655300-worker-0_11071760_b4e541e4-643a-11eb-9823-f403434ab74c_0
```
The app container already exited, `docker logs` can show the training program already started but exited with error.

**What you expected to happen**:
The pod's status should be 'Error' or 'Completed' if the pod already exited.

**How to reproduce it (as minimally and precisely as possible)**:
It happens occasionally in my environment, I have not found the steps to reproduce it.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): v1.14.3
- Cloud provider or hardware configuration: none
- OS (e.g: `cat /etc/os-release`): CentOS 7
- Kernel (e.g. `uname -a`):  3.10.0-693.el7.x86_64
- Install tools: RKE
- Network plugin and version (if this is a network-related bug): 
- Others:

**Comment by chentaoz** [comment]
hey @zionwu , can you share any log in kubelet and scheduler for this issue?

**Comment by zionwu** [comment]
The log of kubelet:
```
I0203 02:35:57.472136  153592 kubelet.go:1885] SyncLoop (ADD, "api"): "sky-segment-20210203103450226-worker-0_11071760(876c2166-65c8-11eb-9823-f403434ab74c)"
I0203 02:35:57.570139  153592 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "default-token-6sk7d" (UniqueName: "kubernetes.io/secret/876c2166-65c8-11eb-9823-f403434ab74c-default-token-6sk7d") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.570184  153592 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "data-1" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-1") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.570207  153592 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "data-0" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-0") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.570299  153592 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "data-2" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-2") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.570376  153592 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "dshm" (UniqueName: "kubernetes.io/empty-dir/876c2166-65c8-11eb-9823-f403434ab74c-dshm") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.570407  153592 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "code-vol" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-code-vol") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.570430  153592 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "platform-vol" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-platform-vol") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.670669  153592 reconciler.go:252] operationExecutor.MountVolume started for volume "data-1" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-1") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.670704  153592 reconciler.go:252] operationExecutor.MountVolume started for volume "data-0" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-0") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.670729  153592 reconciler.go:252] operationExecutor.MountVolume started for volume "data-2" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-2") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.670751  153592 reconciler.go:252] operationExecutor.MountVolume started for volume "dshm" (UniqueName: "kubernetes.io/empty-dir/876c2166-65c8-11eb-9823-f403434ab74c-dshm") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.670895  153592 reconciler.go:252] operationExecutor.MountVolume started for volume "platform-vol" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-platform-vol") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.670943  153592 reconciler.go:252] operationExecutor.MountVolume started for volume "code-vol" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-code-vol") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.671006  153592 reconciler.go:252] operationExecutor.MountVolume started for volume "default-token-6sk7d" (UniqueName: "kubernetes.io/secret/876c2166-65c8-11eb-9823-f403434ab74c-default-token-6sk7d") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.700993  153592 operation_generator.go:669] MountVolume.SetUp succeeded for volume "dshm" (UniqueName: "kubernetes.io/empty-dir/876c2166-65c8-11eb-9823-f403434ab74c-dshm") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.706055  153592 operation_generator.go:669] MountVolume.SetUp succeeded for volume "default-token-6sk7d" (UniqueName: "kubernetes.io/secret/876c2166-65c8-11eb-9823-f403434ab74c-default-token-6sk7d") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.786026  153592 operation_generator.go:669] MountVolume.SetUp succeeded for volume "code-vol" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-code-vol") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.787291  153592 operation_generator.go:669] MountVolume.SetUp succeeded for volume "platform-vol" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-platform-vol") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.797020  153592 operation_generator.go:669] MountVolume.SetUp succeeded for volume "data-2" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-2") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.800317  153592 operation_generator.go:669] MountVolume.SetUp succeeded for volume "data-0" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-0") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:57.856398  153592 operation_generator.go:669] MountVolume.SetUp succeeded for volume "data-1" (UniqueName: "kubernetes.io/glusterfs/876c2166-65c8-11eb-9823-f403434ab74c-data-1") pod "sky-segment-20210203103450226-worker-0" (UID: "876c2166-65c8-11eb-9823-f403434ab74c") 
I0203 02:35:58.080807  153592 kuberuntime_manager.go:409] No sandbox for pod "sky-segment-20210203103450226-worker-0_11071760(876c2166-65c8-11eb-9823-f403434ab74c)" can be found. Need to start a new one
2021-02-03 02:36:03.113 [INFO][196769] plugin.go 75: Extracted identifiers EndpointIDs=&utils.WEPIdentifiers{Namespace:"11071760", WEPName:"", WorkloadEndpointIdentifiers:names.WorkloadEndpointIdentifiers{Node:"10.196.18.3", Orchestrator:"k8s", Endpoint:"eth0", Workload:"", Pod:"sky-segment-20210203103450226-worker-0", ContainerID:"400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea"}}
I0203 02:36:03.119473  153592 kubelet.go:1930] SyncLoop (PLEG): "sky-segment-20210203103450226-worker-0_11071760(876c2166-65c8-11eb-9823-f403434ab74c)", event: &pleg.PodLifecycleEvent{ID:"876c2166-65c8-11eb-9823-f403434ab74c", Type:"ContainerStarted", Data:"400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea"}
2021-02-03 02:36:03.184 [INFO][196769] plugin.go 166: Calico CNI found existing endpoint: &{{WorkloadEndpoint projectcalico.org/v3} {10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0  11071760  876c2166-65c8-11eb-9823-f403434ab74c 2903605448 0 2021-02-03 02:35:50 +0000 UTC <nil> <nil> map[tf_job_name:sky-segment-20210203103450226 projectcalico.org/namespace:11071760 projectcalico.org/orchestrator:k8s projectcalico.org/serviceaccount:default group_name:kubeflow.org tf-replica-index:0 tf-replica-type:worker] map[] [] nil [] } {k8s  10.196.18.3  sky-segment-20210203103450226-worker-0 eth0 [] []   [kns.11071760 ksa.11071760.default] cali15ebe25e674  [{tfjob-port TCP 2222}]}} ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-"
2021-02-03 02:36:03.184 [INFO][196769] k8s.go 60: Extracted identifiers for CmdAddK8s ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
2021-02-03 02:36:03.374 [INFO][196796] ipam_plugin.go 208: Calico CNI IPAM request count IPv4=1 IPv6=0 ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" HandleID="k8s-pod-network.400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Workload="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
2021-02-03 02:36:03.410 [INFO][196796] ipam_plugin.go 220: Calico CNI IPAM handle=k8s-pod-network.400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" HandleID="k8s-pod-network.400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Workload="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
2021-02-03 02:36:03.410 [INFO][196796] ipam_plugin.go 230: Auto assigning IP ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" HandleID="k8s-pod-network.400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Workload="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0" assignArgs=ipam.AutoAssignArgs{Num4:1, Num6:0, HandleID:(*string)(0xc000248bb0), Attrs:map[string]string{"node":"10.196.18.3", "pod":"sky-segment-20210203103450226-worker-0", "namespace":"11071760"}, Hostname:"10.196.18.3", IPv4Pools:[]net.IPNet{}, IPv6Pools:[]net.IPNet{}, MaxBlocksPerHost:0}
2021-02-03 02:36:03.546 [INFO][196796] ipam_plugin.go 232: Calico CNI IPAM assigned addresses IPv4=[10.227.83.170/26] IPv6=[] ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" HandleID="k8s-pod-network.400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Workload="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
2021-02-03 02:36:03.546 [INFO][196796] ipam_plugin.go 258: IPAM Result ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" HandleID="k8s-pod-network.400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Workload="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0" result.IPs=[]*current.IPConfig{(*current.IPConfig)(0xc000452540)}
2021-02-03 02:36:03.547 [INFO][196769] k8s.go 361: Populated endpoint ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0" endpoint=&v3.WorkloadEndpoint{TypeMeta:v1.TypeMeta{Kind:"WorkloadEndpoint", APIVersion:"projectcalico.org/v3"}, ObjectMeta:v1.ObjectMeta{Name:"10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0", GenerateName:"", Namespace:"11071760", SelfLink:"", UID:"876c2166-65c8-11eb-9823-f403434ab74c", ResourceVersion:"2903605448", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:63747916550, loc:(*time.Location)(0x232eae0)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string{"group_name":"kubeflow.org", "tf-replica-index":"0", "tf-replica-type":"worker", "tf_job_name":"sky-segment-20210203103450226", "projectcalico.org/namespace":"11071760", "projectcalico.org/orchestrator":"k8s", "projectcalico.org/serviceaccount":"default"}, Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Initializers:(*v1.Initializers)(nil), Finalizers:[]string(nil), ClusterName:""}, Spec:v3.WorkloadEndpointSpec{Orchestrator:"k8s", Workload:"", Node:"10.196.18.3", ContainerID:"", Pod:"sky-segment-20210203103450226-worker-0", Endpoint:"eth0", IPNetworks:[]string{"10.227.83.170/32"}, IPNATs:[]v3.IPNAT(nil), IPv4Gateway:"", IPv6Gateway:"", Profiles:[]string{"kns.11071760", "ksa.11071760.default"}, InterfaceName:"cali15ebe25e674", MAC:"", Ports:[]v3.EndpointPort{v3.EndpointPort{Name:"tfjob-port", Protocol:numorstring.Protocol{Type:1, NumVal:0x0, StrVal:"TCP"}, Port:0x8ae}}}}
2021-02-03 02:36:03.547 [INFO][196769] k8s.go 362: Calico CNI using IPs: [10.227.83.170/32] ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
2021-02-03 02:36:03.547 [INFO][196769] network_linux.go 76: Setting the host side veth name to cali15ebe25e674 ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
2021-02-03 02:36:03.548 [INFO][196769] network_linux.go 396: Disabling IPv4 forwarding ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
2021-02-03 02:36:03.578 [INFO][196769] k8s.go 388: Added Mac, interface name, and active container ID to endpoint ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0" endpoint=&v3.WorkloadEndpoint{TypeMeta:v1.TypeMeta{Kind:"WorkloadEndpoint", APIVersion:"projectcalico.org/v3"}, ObjectMeta:v1.ObjectMeta{Name:"10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0", GenerateName:"", Namespace:"11071760", SelfLink:"", UID:"876c2166-65c8-11eb-9823-f403434ab74c", ResourceVersion:"2903605448", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:63747916550, loc:(*time.Location)(0x232eae0)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string{"tf-replica-index":"0", "tf-replica-type":"worker", "tf_job_name":"sky-segment-20210203103450226", "projectcalico.org/namespace":"11071760", "projectcalico.org/orchestrator":"k8s", "projectcalico.org/serviceaccount":"default", "group_name":"kubeflow.org"}, Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Initializers:(*v1.Initializers)(nil), Finalizers:[]string(nil), ClusterName:""}, Spec:v3.WorkloadEndpointSpec{Orchestrator:"k8s", Workload:"", Node:"10.196.18.3", ContainerID:"400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea", Pod:"sky-segment-20210203103450226-worker-0", Endpoint:"eth0", IPNetworks:[]string{"10.227.83.170/32"}, IPNATs:[]v3.IPNAT(nil), IPv4Gateway:"", IPv6Gateway:"", Profiles:[]string{"kns.11071760", "ksa.11071760.default"}, InterfaceName:"cali15ebe25e674", MAC:"12:5d:0d:c2:f6:88", Ports:[]v3.EndpointPort{v3.EndpointPort{Name:"tfjob-port", Protocol:numorstring.Protocol{Type:1, NumVal:0x0, StrVal:"TCP"}, Port:0x8ae}}}}
2021-02-03 02:36:03.587 [INFO][196769] k8s.go 420: Wrote updated endpoint to datastore ContainerID="400b4b3610523841a90f1e24a91f3e09db29f71f0715c1e2a8452cb0e9ae20ea" Namespace="11071760" Pod="sky-segment-20210203103450226-worker-0" WorkloadEndpoint="10.196.18.3-k8s-sky--segment--20210203103450226--worker--0-eth0"
I0203 02:36:03.589665  153592 kubelet.go:1892] SyncLoop (UPDATE, "api"): "sky-segment-20210203103450226-worker-0_11071760(876c2166-65c8-11eb-9823-f403434ab74c)"
I0203 02:36:06.152134  153592 kubelet.go:1930] SyncLoop (PLEG): "sky-segment-20210203103450226-worker-0_11071760(876c2166-65c8-11eb-9823-f403434ab74c)", event: &pleg.PodLifecycleEvent{ID:"876c2166-65c8-11eb-9823-f403434ab74c", Type:"ContainerDied", Data:"687aa9d3b85526cdb0f332824f0e6b14f01cf154450f2aa898152220191afa2e"}

```

**Comment by haircommander** [comment]
This bug seems to be happening in an out-of-support version 1.14.3. Please reopen if the behavior is still happening in a supported version
/close

**Comment by cardil** [comment]
/reopen

I'm seeing the same behavior on GKE `1.24.7`. In my case, that's a regular batch/v1 Job's pod, which sometimes have this status. It makes the Job failing, despite the pod successfully completed (the pod execution is quick, sub second).

Described at: https://github.com/knative-sandbox/kn-plugin-event/issues/242#issuecomment-1318558734

**Comment by cardil** [comment]
@haircommander Could you please reopen this issue?

**Comment by SergeyKanzhelev** [comment]
@cardil I see that the linked bug from knative is closed. Does it mean that the issue is not happening any longer? Do you have some repo steps we can use to investigate?

/triage needs-information

**Comment by cardil** [comment]
I think the issue is still lurking somewhere.

For Knative the issue is not present anymore, after we switched our GKE test clusters from using `beta` to ``, means calling `gcloud container clusters create` instead of `gcloud beta container clusters create`. I did pretty sure the issue went away, just by this change.

I don't know what such change means for the resulting Kubernetes cluster (the version is reported the same), but definitely the `beta` clusters were affected.

The issue was always present in specific scenario: Running a job, a quick one, where first job failed due to the network issues, and second pod succeeded despite having state:

```
ContainersNotReady containers with unready status: [kn-event-sender]
```

IDK what are the steps to reproduce this, apart from what I described above. I failed to reproduce this in other ways I've tried.
